ddl/serial_test.go

```diff
@@ -838,6 +835,13 @@ func (s *testSerialSuite) TestAutoRandom(c *C) {
 	assertPKIsNotHandle("create table t (a bigint auto_random(3), b int, primary key (a, b))", "a")
 	assertPKIsNotHandle("create table t (a int auto_random(3), b int, c char, primary key (a, c))", "a")
 
+	// PKIsNotHandle: table is created when alter-primary-key = true.
+	testutil.ConfigTestUtils.EnableAlterPrimaryKeyConfig()
```

Comment:
If `EnableAlterPrimaryKeyConfig` is only used here(test). Could we use `config.GetGlobalConfig().AlterPrimaryKey = true` directly?